dvc/remote/local.py

```diff
@@ -73,6 +74,15 @@ def __init__(self, repo, config):
         else:
             self.cache_types = copy(self.DEFAULT_CACHE_TYPES)
 
+        self.hash = [utils.CHECKSUM_MD5]
+        if repo and repo.config and repo.config.config:
+            conf = repo.config.config[Config.SECTION_HASH]
+            hash_local = conf.get(Config.SECTION_HASH_LOCAL, None)
+            if hash_local:
+                if isinstance(hash_local, str):
+                    hash_local = [h.strip() for h in hash_local.split(",")]
+                self.hash = hash_local
```

Comment:
do we check for supported hashes here? It looks like a code duplicate from the config class that reads a list. Should be abstracted and reused most likely.